### ISSUE TITLE
Documentation build instructions fail using conda

### ISSUE THREAD

**Issue by pavithraes** [dask/dask#8188]
**What happened**: While building the docs using the conda commands here: https://docs.dask.org/en/latest/develop.html#how-to-build-the-dask-documentation, I get the following error

```
PackagesNotFoundError: The following packages are not available from current channels:

  - //github.com/dask/distributed
```

I see that `docs/requirements.txt` was changed in July: https://github.com/dask/dask/pull/7943/files
and I’m assuming `git+https://github.com/dask/distributed` works only with pip?

**What you expected to happen**: Successful conda env build

**Minimal Complete Verifiable Example**:

```python
git clone https://github.com/pavithraes/dask.git
cd dask/docs
conda create -n daskdocs -c conda-forge --file requirements-docs.txt
conda activate daskdocs
```

**Anything else we need to know?**:

**Environment**:

- Operating System: MacOS

**Comment by pavithraes** [comment]
@jsignell pinging you here because you've been working on docs recently :)

cc @ncclementi

**Comment by ncclementi** [comment]
Thank you @pavithraes for reporting this. I was able to reproduce and I added a comment on the Docs reorganization PR since there are some changes in the `requirements-docs.txt` file that will have the same issue.

**Comment by jrbourbeau** [comment]
Just wanted to note that it looks like @jsignell is taking care of this issue over in https://github.com/dask/dask/pull/7984

**Comment by ncclementi** [comment]
Yes, I was thinking we can close it once that PR is merged, since they are related.
